### DISCUSSION TITLE
Make FunOwl available as a plugin

### DISCUSSION THREAD

**Discussion by nicholascar** [RDFLib/rdflib#1551] [Ideas]
An OWL2 Functional Style Syntax parser/reader, based on RDFlib, is available: [FunOwl](https://github.com/hsolbrig/funowl). I propose this becomes a parser/serializer plugin for RDFlib, just as JSON-LD is now.

I don't think it should be a core (included) parser/serializer since it deals with OWL, not just RDF, but it would be great to have it available with nice RDFlib integration.

**Comment by hsolbrig** [comment]
Agree.   How to proceed?

**Comment by nicholascar** [comment]
I had to create an Issue for this so I could raise it in the [6.0.0 Planning Meeting](https://github.com/RDFLib/rdflib/wiki/2020-04-28-6.0.0-Planning-Meeting) and have it listed in the Milestone 6.0.0. 

If you'd like to join the planning meeting to champion this one, great! If you can't make it, no worries, I can just state that this is a proposed work item and we have a champion for it.

The only potentially related plannign item is discussion around "Creating a stable core, with many satellite plugins", as per the meeting agenda, but I think this will be a plugin, like JSON-LD so a satellite, so perhaps nothing will change there.

**Comment by hsolbrig** [comment]
I would like to join the meeting -- where can I find the details?
